README.md

```diff
@@ -1,8 +1,9 @@
 # CISCO ASA Ansible Collection
 
-[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.asa) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/cisco)](https://codecov.io/gh/ansible-collections/cisco.asa)-->
 [![CI](https://github.com/ansible-collections/cisco.asa/actions/workflows/tests.yml/badge.svg?branch=main&event=schedule)](https://github.com/ansible-collections/cisco.asa/actions/workflows/tests.yml)
 
+⚠️ **The cisco.asa collection has been [deprecated](https://forum.ansible.com/t/the-bullhorn-123/PLACEHOLDER) and will reach it's end-of-life on December, 2026. We are no longer accepting new pull requests, except for ones that fix critical bugs or security vulnerabilities. Compatibility with ansible-core>2.17 is not guaranteed.**
```

Comment:
@KB-perByte Bullhorn 165 is out now!

Although I'm not sure if it's better to link to https://forum.ansible.com/t/38961 or directly to https://forum.ansible.com/t/38939 :shrug: 

@gundalow Do you have an opinion on this? Or do you think it doesn't really matter?

Reply:
Link to the specific forum post https://forum.ansible.com/t/deprecation-announcement-for-the-cisco-asa-ansible-collection/38939 so then people have got a place to discuss

Reply:
Looks better to me, too :-)